Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -36,6 +37,36 @@ namespace gdjs {
       _justLoggedIn = false;
     });
 
+    // If the extension is used, register an eventlistener to know if the user is
+    // logged in while playing the game on Liluo.io.
+    // Then send a message to the parent iframe to say that the player auth is ready.
+    gdjs.registerFirstRuntimeSceneLoadedCallback(
+      (runtimeScene: RuntimeScene) => {
+        _authenticationMessageCallback = (event: MessageEvent) => {
+          receiveMessageFromLiluo(runtimeScene, event, {
+            checkOrigin: true,
+          });
+        };
+        window.addEventListener(
+          'message',
+          _authenticationMessageCallback,
+          true
+        );
+        console.info('Notifying parent iframe that player auth is ready.');
```

Comment:
I think you should probably use `logger.info` (and below, and there is a console.error too).

Nit: "auth" => "authentication". 
Nit: "iframe" => "window" (the parent is not an iframe, it's almost certainly a traditional page. Or could be an iframe, but you don't know and can't know that. If there is any iframe, it's is probably you here).